lstchain/reco/utils.py

```diff
@@ -43,8 +44,8 @@
 
 # position of the LST1
 location = EarthLocation.from_geodetic(-17.89139 * u.deg, 28.76139 * u.deg, 2184 * u.m)
-obstime = Time('2018-11-01T02:00')
-horizon_frame = AltAz(location=location, obstime=obstime)
+obstime_mc = Time('2018-11-01T02:00')
```

Comment:
Just remove this.

Why was this added anyway? What is so special about 2018-11-01T02:00 that it shoud be used as time fo mc?